config/routes.rb

```diff
@@ -0,0 +1,26 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+
+  # namespace :api do
+  #   namespace :v1 do
+  #     resources :users, only: [:create] do 
+  #       resources :reservations,  only: [:index, :create, :delete]
+  #     end
+  #     resources :tutors,  only: [:index, :create, :delete]
+  # #   end
+  # end
```

Comment:
- [ ] Kindly, remove this piece of old commented code so the repo's cleanliness is improved. 👍